downstream/titles/controller/controller-user-guide/docinfo.xml

```diff
@@ -1,4 +1,4 @@
-<title>Automation Controller User Guide</title>
+<title>Automation Controller user guide</title>
```

Comment:
Automation Controller is not a proper noun so this should be:

Automation controller administration guide